docs/core/tools/global-tools.md

```diff
@@ -120,39 +120,66 @@ dotnet --list-runtimes
 
 Contact the author of the Global Tool and see if they can recompile and republish their tool package to NuGet with an updated version number. Once they have updated the package on NuGet, you can update your copy.
 
-The .NET Core CLI tries to add the default locations to the PATH environment variable on its first usage. However, there are a couple of scenarios where the location might not be added to PATH automatically, such as:
+The .NET Core CLI tries to add the default locations to the `PATH` environment variable on its first usage. `PATH` isn't updated on macOS if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 
-* If you've set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` environment variable.
+<!-- 
+However, there are a couple of scenarios where the location might not be added to PATH automatically, such as:
+
+* If you've set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` environment variable. 
 * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 * On Linux, you need to edit the shell environment file to configure the PATH.
+Linux is no longer a problem when installed on bash with .deb and .rpm - see https://dotnet.microsoft.com/download/linux-package-manager/ubuntu16-04/sdk-current
```

Comment:
I read this as implying that Linux needs to be installed with .deb and .rpm. It might be clearer to say something like: "This is no longer a problem on Linux if the .NET Core CLI is installed on bash with .deb and .rpm"

Reply:
I always works on bash. There is no change on that part since released